rosdep/python.yaml

```diff
@@ -1505,7 +1505,6 @@ python-espeak:
     trusty: [python-espeak]
     trusty_python3: [python3-espeak]
 python-evdev:
-  gentoo: [dev-python/python-evdev]
```

Comment:
I think we should still keep the non-python3 rule (at least I don't think it's hurting anyone to leave it).

Reply:
But if I understood correctly, this Python 2 rule is then linking to a Python 3 package?

Reply:
I think that's correct. I'm just hesitant removing an existing key. @tfoote Could there be any negative consequences to removing the old entry?

Reply:
We should not remove this rule. This is an intricacy of the Gentoo deployment.

All python keys are inherently the system version on Gentoo now. They're not "incorrect" as such, code that is running on Gentoo is going to run with the system version of python, whichever is selected by the system maintainer. This works with our policy that best policy is that code should be written to be 2/3 compatible if it's 2 compatible. After Noetic and in ROS 2 everything is expected to be python3.